LibTrixi.jl/lib/build.jl

```diff
@@ -54,7 +54,7 @@ header_files = [joinpath(dirname(dirname(@__DIR__)), "src", "trixi.h")]
 # Name of the files that include the initialization functions:
 # - `init.c` contains `trixi_initialize`/`trixi_finalize` for API compatibility
 # - the other file contains the `init_julia`/`shutdown_julia` functions from PackageCompiler
-julia_init_c_file = ["init.c", PackageCompiler.default_julia_init()]
+julia_init_c_file = [joinpath(@__DIR__, "init.c"), PackageCompiler.default_julia_init()]
```

Comment:
Note that PackageCompiler.jl will create the object file `init.o` right next to the input file `init.c`. Thus I think we need to copy the `init.c` first to our build tree somewhere, otherwise we end up polluting the source tree.